Makefile

```diff
@@ -3,6 +3,9 @@
 # See: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
 .PHONY: test test-unit test-e2e images run format verify
 
+ARTIFACT_DIR := $(if $(ARTIFACT_DIR),$(ARTIFACT_DIR),tests/test_results)
```

Comment:
i debated defaulting this to `/dev/null` so you don't get these files unless you explicitly set an env var indicting where you want them to go.

@vishnuchalla @thoraxe  any strong opinions?

otherwise i think this is good to go